### PR TITLE
ci: fix disk issue with crashing osd

### PR DESCRIPTION
**Pull request by leseb**
**Description of your changes:**

Let's try to create a 'ceph' user with the uid 167 and add it to the
'disk' group so that if the permissions on the disk change then the OSD
should be able to access the disk.

Closes: https://github.com/rook/rook/issues/7405
Signed-off-by: Sébastien Han <seb@redhat.com>

**Which issue is resolved by this Pull Request:**
Resolves https://github.com/rook/rook/issues/7405

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
- [ ] **Skip Tests for Docs**: Add the flag for skipping the build if this is only a documentation change. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for the flag.
- [ ] **Skip Unrelated Tests**: Add a flag to run tests for a specific storage provider. See [test options](https://github.com/rook/rook/blob/master/INSTALL.md#test-storage-provider).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.

### CODE REVIEW

**Comment by leseb** [comment]
@satoru-takeuchi the CI got a lot greener with my latest patch, PATL.

File: .github/workflows/integration-test-helm-suite.yaml [hunk 1]
```diff
@@ -35,8 +35,10 @@ jobs:
         sudo swapoff --all --verbose
         sudo umount /mnt
         # search for the device since it keeps changing between sda and sdb
-        sudo wipefs --all --force /dev/$(lsblk|awk '/14G/ {print $1}'| head -1)1
+        PARTITION="/dev/$(lsblk|awk '/14G/ {print $1}'| head -1)1"
+        sudo wipefs --all --force "$PARTITION"
         sudo lsblk
+        echo "SUBSYSTEM==\"block\", ATTR{size}==\"29356032\", ACTION==\"add\", RUN+=\"/bin/chown 167:167 $PARTITION\"" | sudo tee -a /etc/udev/rules.d/01-rook.rules
```
Comment (satoru-takeuchi) [inline .github/workflows/integration-test-helm-suite.yaml:41]: Please write down why this line is necessary in comment.

**Review by satoru-takeuchi — CHANGES_REQUESTED** [review]
In addition to a comment about udev rule, please fix unittest failure.

**Review by satoru-takeuchi — APPROVED** [review]
Please merge it after passing Jenkins CI.

**Comment by leseb** [comment]
> Please merge it after passing Jenkins CI.

It's the Cassandra suite that failed so I believe I can go with it right?

**Comment by satoru-takeuchi** [comment]
Ah, ok. Please merge it right now. I forgot to bypass some tests...

**Pull request merged**